### ISSUE TITLE
Map not rendering in Back Office

### ISSUE THREAD

**Issue by craigs100** [bergmania/Bergmania.OpenStreetMap#27]
Umb V13.2.2 on Linux with SQLite DB

1. Installed OpenStreeMap V5.3.1
2. Created dataType and added to a docType (to replace GMaps package)
3. Regenerated models and rebuilt solution
4. Opened the Location page node and noted the map as a grey block. Console shows the following errors.

![image](https://github.com/bergmania/Bergmania.OpenStreetMap/assets/1143883/fdcc6e4d-7174-4b0a-8b53-1bab28a2558f)

![image](https://github.com/bergmania/Bergmania.OpenStreetMap/assets/1143883/81a42105-9d8a-415d-b86f-229e3c5ad500)

Please advise.

**Comment by craigs100** [comment]
I should say that the V11 test site in the repo runs fine on the same Linux dev machine, so it appears to be an issue with V13.x compatibility.

**Comment by bergmania** [comment]
Hi @craigs100..

I see the problem, but do not think this have ever been a problem in Umbraco. The package do not defend agains unexpected data (e.g the format from GMap, og any other format).

I'll look into if I can figure out a solution - my frontend skills is existing 🙈

**Comment by bergmania** [comment]
Fixed in [5.3.2](https://www.nuget.org/packages/Bergmania.OpenStreetMap/5.3.2) - It will now just show the default map, so the user can change it, if it do not understand the inputted format.

**Comment by craigs100** [comment]
It now appears in the back office, thanks.  However, there are no instructions on how to use it in the site. So wrt OpenStreetMap's Embed code I've taken to doing this:-

```
@{
    Layout = "BasePage.cshtml";

    var boundingBoxModel = Model.OpenStreetMap.BoundingBox;
    var zoom = Model.OpenStreetMap.Zoom;
    var markerModel = Model.OpenStreetMap.Marker;

    var boundingBox = boundingBoxModel.SouthWestCorner.Longitude + "%2C" + boundingBoxModel.SouthWestCorner.Latitude + "%2C" + boundingBoxModel.NorthEastCorner.Longitude + "%2C" + boundingBoxModel.NorthEastCorner.Latitude;
    var marker = markerModel.Latitude + "," + markerModel.Longitude;
}

<iframe width="255" height="280" src="https://www.openstreetmap.org/export/embed.html?bbox=@boundingBox&amp;layer=mapnik&amp;marker=@marker" style="border: 1px solid black"></iframe><br/><small><a href="https://www.openstreetmap.org/?mlat=57.5094&amp;mlon=-5.8633#map=16/57.5094/-5.8633">View Larger Map</a></small>
```
Which sort of works. I can't get the zoom level to work though as OpenStreetMap don't appear to use such a thing. So I'm looking at half of Europe instead of just Scotland.  I had thought there was a property converter that rendered out the necessary HTML?

Please advise.
